.github/workflows/gerrit-compose-required-tox-verify.yaml

```diff
@@ -0,0 +1,92 @@
+---
+name: Compose Tox Verify
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+    inputs:
+      GERRIT_BRANCH:
+        description: "Branch that change is against"
+        required: true
+        type: string
+      GERRIT_CHANGE_ID:
+        description: "The ID for the change"
+        required: true
+        type: string
+      GERRIT_CHANGE_NUMBER:
+        description: "The Gerrit number"
+        required: true
+        type: string
+      GERRIT_CHANGE_URL:
+        description: "URL to the change"
+        required: true
+        type: string
+      GERRIT_EVENT_TYPE:
+        description: "Type of Gerrit event"
+        required: true
+        type: string
+      GERRIT_PATCHSET_NUMBER:
+        description: "The patch number for the change"
+        required: true
+        type: string
+      GERRIT_PATCHSET_REVISION:
+        description: "The revision sha"
+        required: true
+        type: string
+      GERRIT_PROJECT:
+        description: "Project in Gerrit"
+        required: true
+        type: string
+      GERRIT_REFSPEC:
+        description: "Gerrit refspec of change"
+        required: true
+        type: string
+      TARGET_REPO:
+        # yamllint disable-line rule:line-length
+        description: "The target GitHub repository needing the required workflow"
+        required: false
+        default: ${{ github.repository }}
+        type: string
+      TOX_DIR:
+        description: "Directory containing tox.ini file"
+        required: false
+        default: "."
+        type: string
+      TOX_ENVS:
+        description: "Map of versions and envs to run"
+        required: true
+        type: string
+      PARALLEL:
+        description: "Whether to run jobs in parallel"
+        required: false
+        default: "true"
+        type: string
+
+concurrency:
+  # yamllint disable-line rule:line-length
+  group: compose-tox-verify-${{ github.workflow }}-${{ github.event.inputs.GERRIT_CHANGE_ID || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
```

Comment:
Should add a "prepare" or "build" job that builds the package that is then consumed by the tox run itself. This will allow you to refactor the GHA that you've created to not have to deal with dealing with a requirements file. Some projects may have moved completely off of requirements.txt as pyproject.toml becomes more mainstream.

Reply:
Done.